### PR TITLE
Fix ATD specification link

### PR DESCRIPTION
**Pull request by colin4124**

### CODE REVIEW

File: book/json/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ formats/JSON]{.idx #SERjson}[data serialization/with JSON]{.idx #DATjson}
 
 We'll start by using the popular and simple JSON data format and then look at
 other serialization formats later in the book. This chapter introduces you to
-a couple of new techniques that glue together the basic ideas from Part I of
+a couple of new techniques that glue together the basic ideas from Part I off
```
Comment (XVilka) [inline book/json/README.md]: a typo

**Comment by XVilka** [comment]
@colin4124 can you please remove that typo? To make the PR mergeable.

**Comment by colin4124** [comment]
@XVilka Thanks, I have done.

**Comment by XVilka** [comment]
But it is still here.

**Comment by colin4124** [comment]
@XVilka  Should I change "off" to 'of“ ？

**Comment by XVilka** [comment]
Yes, exactly. Now it is fine, thanks!

**Comment by colin4124** [comment]
@XVilka I have send you a email, would you take a look at it? ^_^

**Pull request merged**